test/metabase/api/api_key_test.clj

```diff
@@ -102,3 +103,14 @@
                             :user_id      (mt/user->id :crowberto)
                             :created_by   (mt/user->id :crowberto)}]
         (is (= 2 (mt/user-http-request :crowberto :get 200 "api-key/count")))))))
+
+(deftest api-keys-work-e2e
```

Comment:
Could you add a test that all of the session-level dynamic vars (like `api/is-superuser?`, etc) are bound correctly for API users?